inc/class-sitemaps-pages.php

```diff
@@ -11,6 +11,13 @@ class Core_Sitemaps_Pages extends Core_Sitemaps_Provider {
 	 * @var string
 	 */
 	protected $post_type = 'page';
+	/**
+	 * Sitemap name
+	 * Used for building sitemap URLs.
+	 *
+	 * @var string
+	 */
+	protected $name = 'pages';
```

Comment:
Why can't we just use the actual post type? Seems unneeded to have this extra plural.

Reply:
The name is used for rewrite rules and in urls https://github.com/GoogleChromeLabs/wp-sitemaps/blob/91094a7d6af70533ec766799a0d2e60897e00321/inc/class-sitemaps-registry.php#L87-L88 whereas the post_type is used for post-type queries. 

The index class has a name but not a post-type. Will need more thinking.